scripts/installer/utils/logger.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import sys
+from logging import handlers
+from pathlib import Path
+
+# Assumes structure: .. > scripts > installer > utils > logger.py; with logs folder peer to installer
+top_dir = Path(__file__).resolve().parents[2]
+log_path = Path(f"{top_dir}/logs/logger.log")
```

Comment:
I think you might need something like this to create `logs/logger.log` if it doesn't already exist:
`os.makedirs(os.path.dirname(log_path), exist_ok=True)`